operator/pom.xml

```diff
@@ -32,11 +32,10 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.operator.sdk.version>3.0.4</quarkus.operator.sdk.version>
-        <quarkus.version>2.7.3.Final</quarkus.version>
+        <quarkus.operator.sdk.version>3.0.6</quarkus.operator.sdk.version>
+        <quarkus.version>2.7.5.Final</quarkus.version>
```

Comment:
> Obviously the risk is that the JOSDK might be incompatible. But did we hit that issue already?

Yup, it happened exactly twice so far that we aligned Quarkus version by chance. I do believe that we are not ready yet as keeping the same version will slow down the Keycloak dist into integrating new Quarkus versions.

But happy to hear other opinions @DGuhr @pedroigor

Reply:
Whatever the result of this discussion will be, it can be detached from this PR.
@vmuzikar do you mind to open an issue that can be tackled separately?

Reply:
Sure, created https://github.com/keycloak/keycloak/issues/11015.